### DISCUSSION TITLE
Help: with use anyio as pathlib.Path

### DISCUSSION THREAD

**Discussion by heavyblack1** [agronholm/anyio#808] [Q&A] [answered]
It should be easy call your `Path.cwd()` and glob for files. But on line using `glob()` it crashes saying `'coroutine' object has no attribute 'glob'` and pylance have problem with data types which I don't understand because your abstract method `async def cwd(cls) -> Path:` say it should return it self. 

Please can you correct me how to use your library. Thanks 

```python
from anyio import Path as AsyncPath

cwd:AsyncPath = AsyncPath.cwd()
# Syntax Error according pylance  
# Type "Coroutine[Any, Any, Path]" is not assignable to declared type "Path"
#  "Coroutine[Any, Any, Path]" is not assignable to "Path"

# Syntax Error according pylance
# Cannot access attribute "glob" for class "Coroutine[Any, Any, Path]" 
#  Attribute "glob" is unknown
async for path in cwd.glob('*.txt'):
    ...
```

**Comment by agronholm** [comment, marked as answer]
Have you tried awaiting on `AsyncPath.cwd()`?

**Reply by heavyblack1** [reply]
Well it was more complicated not just await  but i forget i need individual function call `asyncio.run()` because when I call `asyncio.run(main())` other code calling third party async libs was throwing error( `asyncio.run()` cannot be called from a running event loop) and this is why when I tried add await it sad is not await able  which i forget mention. Thanks
